### DISCUSSION TITLE
Issues with RNG seed in NSGA2

### DISCUSSION THREAD

**Discussion by vj2Sheffield** [mdolab/pyoptsparse#295] [Q&A] [answered]
Hi all,

I've been trying to re-run a multi-objective problem using NSGA2 and replicated the experiment several times over. I'm finding that the output files are showing the exact same best population every time, despite either the seed being either set to 0 or with varying integers. Any help with this?

**Comment by ewu63** [comment, marked as answer]
I don't think any of the maintainers of this repo actively use the NSGA2 optimizer, so we won't be able to help. If you find anything, please feel free to open a pull request.
